python/foxglove-sdk/python/examples/geometry/euler_to_quaternion.py

```diff
@@ -1,6 +1,6 @@
 from math import cos, sin
 
-from foxglove.schemas import Quaternion
+from foxglove._foxglove_py.schemas import Quaternion
```

Comment:
can we re-export these, consumers shouldn't be importing anything from `_foxglove_py` directly